Source/DafnyCore/Backends/Python/PythonCodeGenerator.cs

```diff
@@ -28,6 +39,8 @@ class PythonCodeGenerator : SinglePassCodeGenerator {
 
     private const string DafnyRuntimeModule = "_dafny";
     private const string DafnyDefaultModule = "module_";
+    private const string DafnySystemModule = "System_";
+    private static readonly ISet<string> DafnyRuntimeModuleNames = new HashSet<string> { DafnyRuntimeModule, DafnySystemModule, };
```

Comment:
FYI it's only the internal `System_` Dafny module that goes through this code generation logic (see https://github.com/dafny-lang/dafny/blob/master/Source/DafnyRuntime/DafnyRuntimePython/Makefile). There's no such thing as a Dafny module for the runtime itself, so you can just check for the system module.

Reply:
I just removed `DafnyRuntimeModule` from the set, let me know if you'd prefer I remove the whole set.
I think a one-element set with a defined purpose is clearer so that `System_` isn't randomly checked in the file